web/src/portico/landing-page.js

```diff
@@ -14,6 +14,59 @@ export function path_parts() {
     return window.location.pathname.split("/").filter((chunk) => chunk !== "");
 }
 
+const hello_events = function () {
+    function getNewRand(oldRand, max) {
```

Comment:
I believe the variable and function names in this code should be in snake_case instead of camelCase.

Reply:
Thanks! Done